app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+
+const app = express()
+
+app.use(express.json())
+
+const {getTopics}=require('./controllers/topics.controller')
+
+app.get('/api/topics',getTopics)
+
+
+app.use((err, req, res, next) => {
+    console.log(err);
+    res.status(404).send({ msg: 'Not Found' });
+  });
+
+app.all('*', (req, res) => {
```

Comment:
Your invalid link controller should be above you custom error handler - please change this

Reply:
app.use should be below app.all in this file and app.use use shuld be sending back a 500, not a 404. app.js should hit the app.all 404 controller first.